src/configs/price.ts

```diff
@@ -6,21 +6,25 @@ export const DefaultPriceConfig: PriceConfig = {
     {
       name: "Time: <1 Day",
       weight: 1,
+      value: 3600 * 24,
       target: "Price: 100-1000 USD",
     },
     {
       name: "Time: <1 Week",
       weight: 2,
+      value: 3600 * 24 * 7,
       target: "Price: 200-2000 USD",
     },
     {
       name: "Time: <2 Weeks",
       weight: 3,
+      value: 3600 * 24 * 14,
       target: "Price: 300-3000 USD",
     },
     {
       name: "Time: <1 Month",
       weight: 4,
+      value: 3600 * 24 * 30,
```

Comment:
Wouldn't it be more accurate to divide `365.25 / 12` to derive the monthly average?

Reply:
lol If we have to have really strong requirements in exact timeframe, 365.25 / 12 would also be incorrect. we might need to take every 4 year into consideration as well.

Reply:
The reason why I bring this up is because its applicable for (and perhaps more pronounced for accuracy) with "<2 Weeks" as well. I think it applies across the board, and could be particularly useful when we have custom time estimates (crowdsourced from the DevPool e.g. "3 Days")

Reply:
Basically if we calculate time with precision we should be able to pick any time interval (1 day, 3 days, 5.5 days etc) and it should be accurate. Right now we only have like four time options, but in the future if we do figure out how to incentivize bounty hunters to gamble on the time estimate of the bounties, we could get accurate time estimates via averages/medians for bounties which will be unlikely to be one of the preset values you have here.

The only way to solve this and to make it work with any time interval is to calculate it with precision- perhaps unix time style.

Reply:
Oh I understand what you're saying. I agree to make it working with any time inverval. but we will definitely do it on other issues later.